metals/src/main/scala/scala/meta/internal/metals/debug/ClientConfigurationAdapter.scala

```diff
@@ -35,6 +38,33 @@ private[debug] final case class ClientConfigurationAdapter(
     sourceMapper.mappedLineForClient(path, adaptedLine)
   }
 
+  /**
+   * In the DAP specification, the presentationHint of a StackFrame can be
+   * 'normal', 'label' or 'subtle'. Most DAP implementations use 'subtle' to
+   * indicate that a frame is skipped by the debugger. The problem is that
+   * VSCode does not collapse 'subtle' frames, as other DAP clients do.
+   * Instead it collapses 'deemphasize' frames, even if it is not part of the
+   * spec.
+   *
+   * See https://github.com/microsoft/vscode/issues/206801
+   */
+  def adaptStackTraceResponse(result: JsonObject): JsonObject = {
```

Comment:
Could we wrap it in try and return original result otherwise? Looks like there are some things that could fail.